### PR TITLE
Added command to change SSH port

### PR DESCRIPTION
**Pull request by vjFaLk**
- `bench setup ssh-port [port]`

### CODE REVIEW

File: bench/commands/setup.py [hunk 1]
```diff
@@ -84,6 +84,17 @@ def setup_firewall():
 		abort=True)
 	run_playbook('production/setup_firewall.yml')
 
+@click.command('ssh-port')
+@click.argument('port')
+def set_ssh_port(port):
+	"Setup firewall"
```
Comment (shreyasp) [inline bench/commands/setup.py]: @vjFaLk Update the comment ... Bad Copy Paste 💩

File: bench/utils.py [hunk 2]
```diff
@@ -687,9 +687,11 @@ def set_git_remote_url(git_url, bench_path='.'):
...
 		exec_cmd("git remote set-url upstream {}".format(git_url), cwd=app_dir)
 
-def run_playbook(playbook_name):
+def run_playbook(playbook_name, extra_vars=None):
 	if not find_executable('ansible'):
 		print "Ansible is needed to run this command, please install it using 'pip install ansible'"
 		sys.exit(1)
 	args = ['ansible-playbook', '-c', 'local', playbook_name]
+	if extra_vars:
+		args.extend(['--extra-vars=' + json.dumps(extra_vars)])
```
Comment (shreyasp) [inline bench/utils.py]: Keeping it consistent with `install.py` args.extend(['-e', get_extra_vars_json(extra_vars)])

File: bench/commands/setup.py (hunk 1 above)
Comment (vjFaLk) [inline bench/commands/setup.py]: This is just embarrassing 🙈

**Comment by vjFaLk** [comment]
@shreyasp Fixed issues

**Pull request merged**